sky/backends/backend_utils.py

```diff
@@ -2471,7 +2471,10 @@ def stop_handler(signum, frame):
         raise KeyboardInterrupt(exceptions.SIGTSTP_CODE)
 
 
-def validate_schema(obj, schema, err_msg_prefix=''):
+def validate_schema(obj, schema, err_msg_prefix='', skip_none=False):
```

Comment:
When do we want skip_none=False? In cli.py’s admin_deploy()?

Reply:
We may want it in cases where `None` is a valid value for some config we are validating. After thinking about it further, it makes more sense to set the default to `True`, since it is likelier than we want to remove `None` values than not. I've updated to code for this.